### ISSUE TITLE
[Request] Nintendo 3DS Install Firmware Option and boot to firmware

### ISSUE THREAD

**Issue by Danik2343** [TASEmulators/BizHawk#3716]
Since work on the core has begun, I propose to add the ability to install the 3ds firmware and run the firmware.
The official Citra doesn't handle it well, with the firmware.

**Comment by CasualPokePlayer** [comment]
No, this fundementally won't work with the way BizHawk operates.

Also please don't go open issues on an incomplete core port.
